src/main/java/dinkplugin/notifiers/BaseNotifier.java

```diff
@@ -36,8 +38,21 @@ public boolean isEnabled() {
         if (WorldUtils.isIgnoredWorld(world)) {
             return false;
         }
+        return accountPassesConfig();
+    }
+
+    /**
+     * @return whether the player name passes the configured filtered RSNs and the account type is permitted
+     */
+    protected boolean accountPassesConfig() {
         Player player = client.getLocalPlayer();
-        return player != null && settingsManager.isNamePermitted(player.getName());
+        if (player == null) {
+            return false;
+        }
+        if (config.nameFilterMode() != FilterMode.ALLOW && config.deniedAccountTypes().contains(Utils.getAccountType(client))) {
+            return false;
```

Comment:
Should we add a `log.info` here in case a user gets confused why dinks aren't sending for certain accounts?

Reply:
I think that's fine, debugging issues will be very easy when we open the logs and see a bunch of warnings clearly explaining the problem.